src/service/fixed-layer.js

```diff
@@ -147,7 +149,19 @@ export class FixedLayer {
           stylesheet.cssRules, fixedSelectors, stickySelectors);
     }
 
-    this.trySetupSelectorsNoInline(fixedSelectors, stickySelectors);
+    this.trySetupSelectorsNoInline(dev().assertElement(root), fixedSelectors,
+        stickySelectors);
+
+    root.addEventListener(AmpEvents.DOM_UPDATE, event => {
```

Comment:
Before merge, let's wrap this in an experiment (default off) in case this causes unexpected behavior.